test/Test/DecisionSpec.hs

```diff
@@ -25,3 +29,59 @@ test_decisionMonoidMempty = pure $ testProperty "Decision Monoid:Mempty" $ prope
 
 genDecision :: MonadGen m => m Decision
 genDecision = Gen.enumBounded
+
+genTextArr :: MonadGen m => m [Text]
+genTextArr = Gen.list (Range.constant 0 10) $ Gen.text (Range.constant 0 10) Gen.alpha
+
+genText :: MonadGen m => m Text
+genText = Gen.text (Range.constant 0 10) Gen.alpha
+
+genGhcVerArr :: MonadGen m => m [GhcVer]
+genGhcVerArr = Gen.list (Range.constant 0 10) Gen.enumBounded
+
+genCustomPrelude :: MonadGen m => m CustomPrelude
+genCustomPrelude = Prelude
+    <$> Gen.text (Range.constant 0 10) Gen.alpha
+    <*> Gen.text (Range.constant 0 10) Gen.alpha
```

Comment:
I think it's not only `alpha`. Package names can contain characters (uppercase and lowercase), digits, `-`.
Module names can contain characters (uppercase and lowercase), digits and dots.